devel/minipro/Portfile

```diff
@@ -14,9 +14,15 @@ long_description    Opensource tool that aims to create a complete cross-platfor
                     Currently it supports more than 13000 of target devices including \
                     AVRs, PICs as well as a huge number of other microcontrollers and \
                     various BIOSes.
+homepage            https://gitlab.com/DavidGriffith/minipro
```

Comment:
Just as the github portgroup helps simplify ports hosted at github, the gitlab portgroup should be used to simplify ports hosted using gitlab.